src/components/NewspapersLines.vue

```diff
@@ -44,6 +44,11 @@
             {{ newspaper.endYear}}
           </div>
       </div>
+      <div v-if="!isNaN(firstIssueYear(newspaper)) && !isNaN(lastIssueYear(newspaper))" class="line" :style="{
```

Comment:
I' rewrite it as:
```html
<div v-if="newspaper.firstIssue" class="line included" :style="newspaperIncludedStyle(newspaper)"/>
```
where `newspaperIncludedStyle` would be a method returning `{left, right}`